Scheduler/Services/WeatherService/WeatherService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Scheduler.Models;
+
+namespace Scheduler.Services.WeatherService
+{
+    public class WeatherService : IWeatherService
+    {
+        public async Task<WeatherRoot> GetCurrentWeather(string city)
+        {
+            WeatherRoot currentWeather = null;
```

Comment:
did you tested app without internet? when currentWeather will return as null, does it will be handled?